malasakit-django/pcari/templates/personal-information.html

```diff
@@ -73,7 +74,7 @@
       <td>{% trans 'Province' %}</td>
       <td>
         <select id="province">
-          <option value="" selected>{% trans 'Select province' %}</option>
```

Comment:
Put back this blank value. If no value is provided, the `$(...).val()` is the contents of the option (that is, "Select province"), which breaks the JavaScript (since there may be multiple translations of "Select province", we need a single value to denote no selected province).